src/frontend/skipToken.sk

```diff
@@ -151,15 +138,29 @@ fun escapeStringLiteralValue(s: String): String {
   s.chars().map(escapeCharLiteralValue).join("")
 }
 
-fun accumulateStringValue(acc: String, value: String, index: Int): String {
-  if (index == value.length()) {
-    acc
-  } else {
-    length = charLiteralLength(value, index);
-    accumulateStringValue(
-      acc + charLiteral(value.sub(index, length)),
-      value,
-      index + length,
-    )
-  }
+fun stringValue(value: String): String {
+  chars = value.chars();
```

Comment:
You probably want to do a quick scan first to see if there's a \ and if not, just return the existing string. Otherwise you end up rebuilding strings all the time even though most strings don't actually need to.